lib/radiator/directory.ex

```diff
@@ -30,9 +30,19 @@ defmodule Radiator.Directory do
     queryable
   end
 
-  # TODO: add a notion of published to networks as well and only show those
+  defp published_networks_query do
+    published_podcasts_query = PodcastQuery.filter_by_published(Podcast)
+
+    network_query =
+      from(p in published_podcasts_query, join: n in assoc(p, :network), distinct: n, select: n)
+
+    from(n in subquery(network_query), order_by: [desc: n.title])
+  end
+
+  # TODO: have a clearer concept of published for networks (currently just defers to the podcast)
```

Comment:
I've been thinking about this before and I wonder if we can get away without a network publish state. Your deferrence to podcasts seems fine. Do we ever need a network be public when there is no public podcast inside?

Reply:
Only in terms of a pre-announcement. Which usually is done by publishing an 0-episode, so all of this might be fine.